packages/jest-mock/src/index.js

```diff
@@ -676,6 +686,10 @@ class ModuleMockerClass {
     this._spyState.forEach(restore => restore());
     this._spyState = new Set();
   }
+
+  _typeOf(value: any): string {
```

Comment:
Maybe use `jest-get-type` here?

Reply:
Still seems weird to create a poor-man's version of a get-type when the project includes a util for not. Not a big deal of course 🙂